adapter/internal/oasparser/config_generator.go

```diff
@@ -207,19 +207,27 @@ func GetEnforcerAPI(mgwSwagger model.MgwSwagger, lifeCycleState string, vhost st
 	extendedOperations := []*api.ExtendedOperation{}
 	for _, extOp := range mgwSwagger.GetExtendedOperations() {
 		proxyMapping := &api.ProxyMapping{}
+		backednMapping := &api.BackendMapping{}
 		if extOp.ProxyMapping != nil {
 			proxyMapping.Name = extOp.ProxyMapping.Name
 			proxyMapping.Context = extOp.ProxyMapping.Context
 			proxyMapping.Version = extOp.ProxyMapping.Version
 			proxyMapping.Target = extOp.ProxyMapping.Target
 			proxyMapping.Verb = extOp.ProxyMapping.Verb
 		}
+		if extOp.BackendMapping != nil {
+			backednMapping.Endpoint = extOp.BackendMapping.Endpoint
```

Comment:
should we check for malformed URLs at this point? User would see as their MCP API is deployed but actually it would end up in runtime error.